bagit-profile.json

```diff
@@ -1 +1 @@
-{"BagIt-Profile-Info":{"BagIt-Profile-Identifier":"https://ocr-d.github.io/bagit-profile.json","Source-Organization":"OCR-D","External-Description":"BagIt profile for OCR data","Contact-Name":"Konstantin Baierer","Contact-Email":"konstantin.baierer@sbb.spk-berlin.de","Version":0.1},"Bag-Info":{"Bagging-Date":{"required":false},"Source-Organization":{"required":false},"Ocrd-Mets":{"required":false,"default":"data/mets.xml"},"Ocrd-Manifestation-Depth":{"required":false,"default":"partial","values":["partial","full"]},"Ocrd-Identifier":{"required":true},"Ocrd-Checksum":{"required":false,"default":"cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e"}},"Manifests-Required":["sha512"],"Tag-Manifests-Required":[],"Tag-Files-Required":[],"Allow-Fetch.txt":false,"Serialization":"required","Accept-Serialization":"application/zip","Accept-BagIt-Version":["1.0"]}
\ No newline at end of file
+{"BagIt-Profile-Info":{"BagIt-Profile-Identifier":"https://ocr-d.github.io/bagit-profile.json","Source-Organization":"OCR-D","External-Description":"BagIt profile for OCR data","Contact-Name":"Konstantin Baierer","Contact-Email":"konstantin.baierer@sbb.spk-berlin.de","Version":0.1},"Bag-Info":{"Bagging-Date":{"required":false},"Source-Organization":{"required":false},"Ocrd-Mets":{"required":false,"default":"data/mets.xml"},"Ocrd-Manifestation-Depth":{"required":false,"default":"partial","values":["partial","full"]},"Ocrd-Identifier":{"required":true},"Ocrd-Checksum":{"required":false,"default":"cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e"}},"Manifests-Required":["sha512"],"Tag-Manifests-Required":[],"Tag-Files-Required":[],"Allow-Fetch.txt":true,"Serialization":"required","Accept-Serialization":"application/zip","Accept-BagIt-Version":["1.0"]}
```

Comment:
Why does this file consist of only one line? So the changes are difficult to trace.

Reply:
It's generated from YAML and minified to prevent accidentital editing.